src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -846,6 +849,14 @@ public async Task EnableCollectionEnhancements(Guid id)
 
         organization.FlexibleCollections = true;
         await _organizationService.ReplaceAndUpdateCacheAsync(organization);
+
+        // Force a vault sync for all owners and admins of the organization so that changes show immediately
+        // Custom users are intentionally not handled as they are likely to be less impacted and we want to limit simultaneous syncs
+        var orgUsers = await _organizationUserRepository.GetManyByOrganizationAsync(id, null);
```

Comment:
Previously this used a db call that returned `OrganizationUserUserDetails`, which includes a bunch of unnecessary information from the `User` table. This is a simpler call as it just returns `OrganizationUser` which is a single table. It doesn't filter by `minStatus` so we need to filter the results ourselves but I still think that's likely to be more efficient.